swift.html.markdown

```diff
@@ -74,11 +76,17 @@ if someOptionalString != nil {
     if someOptionalString!.hasPrefix("opt") {
         print("has the prefix")
     }
-    
+
     let empty = someOptionalString?.isEmpty
 }
 someOptionalString = nil
 
+/*
+    To get the underlying type from an optional, you unwrap it using the 
+    force unwrap operator (!). Only use the unwrap operator if you're sure 
+    the underlying value isn't nil.
```

Comment:
I don't think we should put such a statement. We're basically telling "always use `(!)`", which definitely isn't what we want to. the "if let" should look like a good solution as well